src/test/rpc/Tx_test.cpp

```diff
@@ -528,6 +547,29 @@ class Tx_test : public beast::unit_test::suite
                 client.status.ok(), client.reply);
         };
 
+        auto grpcAccountTx = [&grpcPort](
+                                 uint256 const& hash,
```

Comment:
The parameter is called `hash`, but the argument is called `id`. I prefer `id`: a hash is a function that returns a digest which is used as an identifier.